initiatives/submit-a-request/index.html

```diff
@@ -0,0 +1,371 @@
+                           type="text">
+                </div>
+            </fieldset>
+
+            <div class="form-l_col form-l_col-1">
+                <label for="foia-request_request-desc">
+                    <span class="form-label-header">
+                        Request description
+                    </span><br/>
+                    <span class="micro-copy micro-copy__large">
+                        Describe the records you seek from the CFPB. Review our FAQs for additional guidance
+                    </span>
+                </label>
+                <textarea class="input__long" id="foia-request_request-desc"  required>
+                </textarea>
+            </div>
+
+            <div class="form-l_col form-l_col-1 ">
+                <label class="form-label-header" for="foia-request_request-fee">
+                    Select a request category
+                </label>
+                <div class="custom-select">
+                    <select class="custom-select_select"
+                            data-placeholder="Select the FOIA request category"
+                            id="foia-request_request-fee">
+                        <option value="">Select the FOIA request category</option>
+                        <option value="Commercial">Commercial</option>
+                        <option value="Media">Media</option>
+                        <option value="Others">Others</option>
+                    </select>
+                    <span class="custom-select_icon cf-icon cf-icon-down"></span>
+                </div>
+            </div>
+
+            <fieldset class="u-reset js-additional-field">
+                <div class="form-l_col form-l_col-1 u-mb0">
+                    <legend class="form-label-header">
+                        Requesting a fee waiver?
+                    </legend>
+                </div>
+                <div class="form-l_col form-l_col-1-2 form-l-inset">
+                    <div class="form-l-inset_container">
+                        <label>
+                            <input class="custom-input" type="radio" name="request-fee-waiver_button">
+                            No
+                        </label>
+                    </div>
+                </div>
+                <div class="form-l_col form-l_col-1-2 form-l-inset">
+                    <div class="form-l-inset_container">
+                        <label>
+                            <input class="custom-input js-additional-field_trigger"
+                                   type="radio"
+                                   name="request-fee-waiver_button"
+                                   data-additional-fields="#foia-request-fee-waiver-justify">
+                            Yes
+                        </label>
+                    </div>
+                </div>
+            </fieldset>
+
+            <div class="form-l_col form-l_col-1 input__long" id="foia-request-fee-waiver-justify">
```

Comment:
Unused inputs should be set as `disabled` to avoid cf_formValidator from checking them and updated in `toggleAdditionalFields`.